books/RayTracingInOneWeekend.html

```diff
@@ -2884,6 +2936,15 @@
 In service of this, we'll create a new vector method -- `vec3::near_zero()` -- that returns true if
 the vector is very close to zero in all dimensions.
 
+We'll need to use the C++ standard library function `std::fabs`, which returns the absolute value of
+its input. We'll add this to `rtweekend.h` since we'll need this in several locations as we proceed.
+
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
+    XXXX
```

Comment:
Missing code snippet?

Reply:
Oops. Other work in progress that slipped in here.

Reply:
Actually, I'll bring this forward to resolve here.

Reply:
Done.